### PR TITLE
Add CONTRIBUTING.md and issue templates.

### PR DESCRIPTION
**Pull request by sayeaud**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Qlik Platform Examples contributing guide
+
+New platform examples or improvements to existing examples are always welcome. 
+
+- [Code of conduct](#code-of-conduct)
+- [Bugs](#bugs)
+- [Proposing a change](#features)
+- [Developing](#developing)
```
Comment (aihazm) [inline .github/CONTRIBUTING.md:8]:
```suggestion
```

Comment (aihazm) [reply .github/CONTRIBUTING.md:8]: can be removed - no such section

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+## <a name="features"></a> Proposing a change
+
+If you want to propose changes to this project, let us know by [filing a issue](https://github.com/qlik-oss/qlik-platform-examples/issues/new/choose).
+
+## <a name="git"></a> Git Guidelines
+
+Generally, development should be done directly towards the `main` branch.
+
+### Branching
```
Comment (aihazm) [inline .github/CONTRIBUTING.md:28]: in some cases, ex: test in CI that requires secrets - this branching strategy is not ideal...but for the scripts, It should be fine

**Review by aihazm — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (sayeaud) [reply .github/CONTRIBUTING.md:27]: Is there a better way to do this that you can point me at? We don't have secrets now, but we will at some point...

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+   git push -u myfork my-feature-or-bugfix
+   ```
+
+1. Create a Pull Request
+
+   > Before creating a Pull Request, make sure to sign the [CLA](#cla)
+
+   In the Github UI of your fork, create a Pull Request to the `main` branch of the main repository.
+
+   If the branch has merge conflicts or has been outdated, please do a rebase against the `main` branch.
```
Comment (aihazm) [inline .github/CONTRIBUTING.md:63]:
```suggestion
1. Create a branch

   The branch should be based on the `main` branch in the master repository.

   ```sh
   git checkout -b my-feature-or-bugfix main
   ```

1. Commit changes on your branch

   Commit changes to your branch, following the commit message format.

   ```sh
   git commit -m "fix: properly formatted SET statements."
   ```

1. Push the changes

   ```sh
   git push
   ```

1. Create a Pull Request

   > Before creating a Pull Request, make sure to sign the [CLA](#cla)

   In the Github UI, create a Pull Request to the `main` branch of the main repository.

   If the branch has merge conflicts or has been outdated, please do a rebase against the `main` branch.
```

**Pull request merged**